Description:  Don't try and build givaro.
 Upstream always builds it against built mpir, but we want to use the Debian
 package.
Author: Doug Torrance
Last-Update: 2017-06-21

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -950,10 +950,6 @@
 # We always build glpk because it depends on mpir (as gmp), which we always build, and we don't want to
 # have the static version of mpir in memory along with the dynamic version of gmp.
 
-BUILD_givaro=yes
-BUILD_ALWAYS="givaro $BUILD_ALWAYS"
-# givaro uses gmp, but we replace gmp with mpir (a patched version), so we must always build givaro
-
 if [[ $BUILD_mpir = yes ]]
 then AC_MSG_NOTICE([mpir is being built (to replace gmp), so we will build pari])
      BUILD_pari=yes
